tests/Integration/Http/ResourceTest.php

```diff
@@ -167,6 +168,31 @@ public function testResourcesMayHaveOptionalValues()
         ]);
     }
 
+    public function testResourcesMayDosntHaveOptionalValues()
```

Comment:
Unreadable function name
MayDosntHave
- May Doesn't have
- May Have?
- Doesn't Have?

Reply:
Sorry but I’m a little weak in English Can you correct it for me?

Reply:
```suggestion
    public function testResourcesMayHaveOptionalValues()
```

Reply:
```suggestion
    public function testResourcesMayHaveOptionalValues()
```